### PR TITLE
LPS-96242 Add checkFormat and lint to test

### PR DESCRIPTION
**Pull request by boton**

### CODE REVIEW

File: modules/apps/document-library/document-library-preview-image/npmscripts.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	preset: 'liferay-npm-scripts/src/presets/standard/index',
+	format: [
+		...standardPreset.format,
+		'test/**/*.js',
+	],
+	lint: [
+		...standardPreset.lint,
+		'test/**/*.js',
+	]
+};
```
Comment (wincent) [inline modules/apps/document-library/document-library-preview-image/npmscripts.config.js]: As discussed via Slack, these are the tickets where we are going to make this standard:

- https://issues.liferay.com/browse/LPS-95744
- https://github.com/liferay/liferay-npm-tools/issues/95

For now though, it's appropriate to add this config. We'll remove it later on as part of those tickets.

Comment (boton) [reply modules/apps/document-library/document-library-preview-image/npmscripts.config.js]: Sound good to me ❤️

**Comment by julien** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/73954. [See changes here.](https://github.com/julien/liferay-portal/compare/9b6d0dd5f9ab~2...9b6d0dd5f9ab)

**Pull request closed without merging**